src/pages/Room/Calendar.jsx

```diff
@@ -9,79 +9,70 @@ import { withAuthContext } from "hocs";
 import { NotificationProvider, ModalFormProvider } from "providers";
 import { NoteCard } from "components/NoteCard";
 
-class CalendarComponent extends React.Component {
-  state = {
-    isDBEmpty: false,
-    isDrawerOpen: false,
-    isLoading: true
-  };
+const CalendarPage = props => {
+  const [isDBEmpty, setIsDBEmpty] = React.useState(false);
+  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(true);
 
-  handleDrawerOpen = () => {
-    this.setState({ isDrawerOpen: true });
+  const handleDrawerOpen = () => {
+    setIsDrawerOpen(true);
   };
 
-  handleDrawerClose = () => {
-    this.setState({ isDrawerOpen: false });
+  const handleDrawerClose = () => {
+    setIsDrawerOpen(false);
   };
 
-  handleDBEmpty = () => {
-    this.setState({ isDBEmpty: true });
+  const handleDBEmpty = () => {
+    setIsDBEmpty(true);
   };
 
-  componentDidMount() {
-    this.setState({ isLoading: false });
+  React.useEffect(() => {
```

Comment:
Could we start importing all the useEffect and useState without the dot notation, importing them in the top of the file? I think it would be cleaner